src/anim/controller/anim-controller.js

```diff
@@ -94,8 +94,8 @@ class AnimController {
     }
 
     get playable() {
-        var playable = true;
-        var i;
+        let playable = true;
+        let i;
```

Comment:
If `let i` is defined outside of a for-loop, then it is in outside scope, so re-defining it will lead to an exception.
Please use this common within engine style
`for(let i = 0; ...`

Reply:
With let/const usage, variable hoisting is not a problem anymore, that's why, please don't define variables for loop counters in a header, but use local variables in loops.

Reply:
Updated, thanks!

Reply:
Please, take a look at the whole PR, there are a lot of the same cases.